remoulade/broker.py

```diff
@@ -48,28 +48,38 @@
 #: The global broker instance.
 global_broker: "Optional[Broker]" = None
 
-#: The order in which middlewares are sorted
-middleware_order = [
-    WorkerThreadLogging,
-    AgeLimit,
-    TimeLimit,
-    ShutdownNotifications,
-    Pipelines,
-    Results,
-    CatchError,
-    Retries,
-    CurrentMessage,
-    LoggingMetadata,
-    MaxMemory,
-    MaxTasks,
-    MessageState,
-    Cancel,
-]
-
 #: A list of extra middleware that will be added to every new instance of Broker
 extra_default_middleware: "List[Middleware]" = []
 
 
+def _get_middleware_order():
```

Comment:
I had to transform the global list into a function because opentelemetry.instrumentation.remoulade imports remoulade so I had a circular import. :grimacing:

Reply:
This is a bit ugly, but i guess its ok. Please update changelog file